common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2655,6 +2657,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.SECONDS)
       .createWithDefaultString("5s")
 
+  val HA_MASTER_RATIS_LEADER_ELECTION_MEMBER_MAJORITY_ADD: ConfigEntry[Boolean] =
```

Comment:
Maybe default as true is more reasonable then false, and do we need guarantee this value should be true? seems majority-add default is false in ratis.

Reply:
> Maybe default as true is more reasonable than false
Have set the default value to true.

> do we need guarantee this value should be true?

Yes,  I have checked it in 
```
private def ensureLeaderElectionMemberMajorityAddEnabled[T](master: Master)(f: => T): T = {
    ensureMasterIsLeader(master) {
      if (!master.conf.hasMasterRatisLeaderElectionMemberMajorityAdd) {
        throw new BadRequestException(s"This operation can only be done when" +
          s" ${CelebornConf.HA_MASTER_RATIS_LEADER_ELECTION_MEMBER_MAJORITY_ADD.key} is true.")
      }
      f
    }
  }
```

>  seems majority-add default is false in ratis.
Yes.